src/lib.rs

```diff
@@ -568,17 +579,17 @@ fn parse_sdp_line(line: &str, line_number: usize) -> Result<SdpLine, SdpParserEr
         }
         Some(v) => {
             let trimmed = v.trim();
-            if trimmed.is_empty() {
+            if trimmed.is_empty() && line_type.as_str() != "s" {
```

Comment:
Lets add a comment here that we're specifically allowing an empty s= line because of Chrome.